docs/cryptodoc/src/05_06_hss_lms.rst

```diff
@@ -0,0 +1,416 @@
+.. _pubkey/hss_lms:
+
+HSS/LMS
+=======
+
+Botan implements the Hierarchical Signature System (HSS) with Leighton-Micali
+Hash-Based Signatures (HSS/LMS) as defined in [RFC8554]_. It supports the
+parameter sets defined in [RFC8554]_ and those in [draft-fluhrer-11]_.
+
+Algorithm Internals
+-------------------
+
+HSS/LMS consists of three building blocks. Like most hash-based signature schemes,
+it uses a One-Time Signature (OTS) at its base, named Leighton-Micali OTS
+(LM-OTS). The public keys of multiple LM-OTS instances are processed to the leaves
+of a Merkle tree. This composition is called the Leighton-Micali Signature (LMS)
```

Comment:
I didn't want to say "... are the public keys" because they were hashed before. But I agree that "are processed" sounds somehow weird. What do you think of `The public keys of multiple LM-OTS instances compose the leaves of a Merkle tree.`?

```

Reply:
Sounds better to me